### ISSUE TITLE
video-crop behavior if lavfi-crop filter is set

### ISSUE THREAD

**Issue by Ashyni** [mpv-player/mpv#12587]
### Important Information

- mpv version: master 779888136018adcf058d8578fc596eabb55a52f4
- Platform and Version: Win 11
- Source of the mpv binary: build with shinchiro/mpv-winbuild-cmake

### Reproduction steps
If we use a recent build, video-crop spam logs if lavfi-crop filter is set, also the Stats screen show `Output Resolution: 1920 x 804, 2.39:1 (Panavision)`

With older build from 3 sept mpv-x86_64-v3-20230903-git-67368ac for example, there are no logs about video-crop and the crop was also set on the Stats screen `Output Resolution: 1920 x 804, 2.39:1 (Panavision), [x: 0, y: 0, w: 1920, h: 804]`

test.lua
```
local function on_start() mp.commandv("vf", "append", "@test:lavfi-crop=w=1920:h=804:x=0:y=138") end
mp.register_event("file-loaded", on_start)
```

### Expected behavior
Fewer logs

### Actual behavior
A lot of logs from video-crop, no functional issue to report.

### Log file
```
[   0.689][d][ffmpeg/video] h264: Decoding frame, 69 bytes, 1 slices
[   0.691][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.691][v][cplayer] Set property: video-crop -> 1
[   0.693][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.693][v][cplayer] Set property: video-crop -> 1
[   0.693][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.693][v][cplayer] Set property: video-crop -> 1
[   0.694][v][vo/gpu-next/libplacebo] Estimated source FPS: 23.923, display FPS: 23.952
[   0.697][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.697][v][cplayer] Set property: video-crop -> 1
[   0.697][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.697][v][cplayer] Set property: video-crop -> 1
[   0.707][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.707][v][cplayer] Set property: video-crop -> 1
[   0.707][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.707][v][cplayer] Set property: video-crop -> 1
[   0.717][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.717][v][cplayer] Set property: video-crop -> 1
[   0.718][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.718][v][cplayer] Set property: video-crop -> 1
[   0.727][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.727][v][cplayer] Set property: video-crop -> 1
[   0.728][d][ffmpeg/video] h264: Decoding frame, 69 bytes, 1 slices
[   0.732][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.732][v][cplayer] Set property: video-crop -> 1
[   0.735][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.735][v][cplayer] Set property: video-crop -> 1
[   0.735][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.735][v][cplayer] Set property: video-crop -> 1
[   0.737][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.737][v][cplayer] Set property: video-crop -> 1
[   0.738][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.738][v][cplayer] Set property: video-crop -> 1
[   0.747][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.747][v][cplayer] Set property: video-crop -> 1
[   0.748][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.748][v][cplayer] Set property: video-crop -> 1
[   0.757][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.757][v][cplayer] Set property: video-crop -> 1
[   0.758][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.758][v][cplayer] Set property: video-crop -> 1
[   0.767][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.768][v][cplayer] Set property: video-crop -> 1
[   0.768][w][vo/gpu-next] Ignoring invalid --video-crop= for 1920x804 image
[   0.768][v][cplayer] Set property: video-crop -> 1
[   0.777][d][ffmpeg/video] h264: Decoding frame, 69 bytes, 1 slices
```

**Comment by forthrin** [comment]
I suggest that `--video-crop` silently nudges down to the available space rather than flooding about value overflow.

**Comment by Dudemanguy** [comment]
Fixed by #12590. Github is too stupid to accept a comma separated issue list.
